chatbot-langchain/ai-studio.yaml

```diff
@@ -15,11 +15,13 @@ application:
         - amd64
       ports:
         - 8001
+      image: quay.io/redhat-et/llamacpp-server
```

Comment:
Currently, the model-server built w/ gh actions is [redhat-etlocallm-model-service](https://quay.io/repository/redhat-et/locallm-model-service) - so should be `quay.io/redhat-et/locallm-model-service:latest`

Reply:
my bad, i didn't check the gh actions. Fixed 👍 Thanks!!